formula-android/src/main/java/com/instacart/formula/android/FeatureView.kt

```diff
@@ -16,12 +16,12 @@ import io.reactivex.rxjava3.core.Observable
  * @param bind A bind function connects state observable to the view rendering.
  * @param lifecycleCallbacks Optional lifecycle callbacks if you need to know the Fragment state.
  */
-class FeatureView<RenderModel>(
+class FeatureView<RenderModel : Any>(
```

Comment:
It would not compile without adding this in various places, unfortunately. I believe this stems from RxJava (hence why I updated the dependency) but I am not 100% sure.